packages/playwright-test/src/loader.ts

```diff
@@ -15,7 +15,8 @@
  */
 
 import { installTransform, setCurrentlyLoadingTestFile } from './transform';
-import type { FullConfig, Config, FullProject, Project, ReporterDescription, PreserveOutput } from './types';
+import type { Config, FullProject, Project, ReporterDescription, PreserveOutput } from './types';
+import type { FullConfigInternal as FullConfig } from './types';
```

Comment:
Let's update the code to explicitly use FullConfigInternal in the signature and avoid possible confusion?